ballot.sol.js

```diff
@@ -0,0 +1,68 @@
+var multi = function(func) { return func.toString().match(/[^]*\/\*([^]*)\*\/\}$/)[1]; }
```

Comment:
yeah it certainly is a hack, but the only way i could remove it from index.html, without having to concat tonnes of strings together.